pytest_mh/utils/fs.py

```diff
@@ -230,6 +231,23 @@ def exists(self, path: str) -> bool:
 
         return False
 
+    def selinux_context(self, path: str) -> SELinuxContext | None:
```

Comment:
If this can be None, we will always need to run two asserts
assert x in not None, "Could not detect selinux context"
assert x.type == "foo_t", "Context does not match"
 
I would rather see the object of the type SELinuxContext that has "bogus values" that You can just directly compare instead.

Reply:
Returning both SELinuxContext or None is generally preferred in Python for its clarity and flexibility in handling the potential absence of a value. It could happen that the command didn't run successfully and we don't get any SELinux context, thus checking for it is the only way of getting the proper verbosity output in the logs. I could also return an exception, but that wouldn't be as clear as None.